src/com_tjvendors/site/includes/vendor.php

```diff
@@ -660,7 +661,15 @@ public function getVendorProfileStatus($userId, $client)
 		$total += (!empty($data->vat_number)) ? 5: 0;
 		$total += (!empty($data->vendor_description)) ? 5: 0;
 		$total += (!empty($data->vendor_logo))? 5: 0;
-		$total += (!empty($data->payment_gateway)) ? 5: 0;
+
+		if ($sendPaymentsToOwner == '0')
```

Comment:
@deepgawade89 you want to check 2 things here. First - send payment directly to the vendor is enabled or not.
OR
Second - Adaptive/split payment gateway is getting used by the client (JT/JGive)